node/module.ts

```diff
@@ -607,6 +608,10 @@ nativeModulePolyfill.set(
   "constants",
   createNativeModule("constants", nodeConstants),
 );
+nativeModulePolyfill.set(
+  "child_process",
+  createNativeModule("child_process", nodeChildProcess),
+);
```

Comment:
Could you update the corresponding test case of this change in module_test.ts for consistency?

Reply:
Done!